### PR TITLE
Improvement to Ansible Tower template runner.

### PR DESCRIPTION
**Pull request by lfu**
Blocks https://github.com/ManageIQ/manageiq-content/pull/528.

https://bugzilla.redhat.com/show_bug.cgi?id=1623636

@miq-bot add_label enhancement, hammer/no, changelog/yes

### CODE REVIEW

**Comment by lfu** [comment]
@miq-bot add_reviewer @bzwei

File: app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def start
     time = Time.zone.now
     update_attributes(:started_on => time)
     miq_task.update_attributes(:started_on => time)
-    my_signal(false, :launch_ansible_tower_job)
+    my_signal(false, :launch_ansible_tower_job, :priority => MiqQueue::HIGH_PRIORITY)
```
Comment (bzwei) [inline app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb]: can we default to normal priority, and accept an option to increase priority?

Comment (bzwei) [reply app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb]: Also while the start may need to be in high priority (caller is waiting to get a task handle), the retry/wait_for_completion can in normal priority.

Comment (lfu) [reply app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb]: That is how it is implemented.

File: app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb [hunk 2]
```diff
@@ -102,16 +113,16 @@ def load_transitions
...
     else
-      queue_signal(action, *args, :deliver_on => deliver_on)
+      queue_signal(action, *args, :deliver_on => deliver_on, :priority => priority)
     end
   end
 
-  def queue_signal(*args, deliver_on: nil)
-    priority = options[:priority] || MiqQueue::NORMAL_PRIORITY
+  def queue_signal(*args, deliver_on: nil, priority: nil)
+    priority ||= options[:priority] || MiqQueue::NORMAL_PRIORITY
```
Comment (lfu) [inline app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb]: @bzwei The default priority is normal.

File: app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb (hunk 1 above)
Comment (bzwei) [reply app/models/manageiq/providers/ansible_tower/automation_manager/template_runner.rb]: I mean let the caller to decide the priority when call `start` or `create_job`, not hardcode the `HIGH_PRIORITY` here.

**Comment by bzwei** [comment]
@lfu Please fix CI build failure

**Pull request merged**